src/components/Zap/ZapAction/ZapAction.tsx

```diff
@@ -0,0 +1,288 @@
+                    >
+                      <ContentCopyIcon sx={{ width: '16px', height: '16px' }} />
+                    </ZapActionProtocolShare>
+                  )}
+                </Box>
+              </ZapActionProtocolCard>
+            </ZapActionProtocolIntro>
+            {detailInformation?.faqItems && (
+              <ZapActionProtocolCard sx={{ padding: '20px 12px' }}>
+                <AccordionFAQ
+                  showIndex={true}
+                  showDivider={true}
+                  showAnswerDivider={true}
+                  sx={{
+                    padding: 0,
+                    '> .accordion-items div:hover': {
+                      background: '#F9F9F9',
+                    },
+                  }}
+                  itemSx={{
+                    padding: '0px 8px',
+                    backgroundColor: 'transparent',
+                    '.MuiAccordionSummary-root': {
+                      padding: 0,
+                    },
+                    '.accordion-items': {
+                      gap: '4px',
+                    },
+                    '.MuiAccordionDetails-root': {
+                      padding: '20px 16px 16px',
+                    },
+                    // '& > div': {
+                    //   borderTop: `1px solid ${alpha(theme.palette.text.primary, 0.04)}`,
+                    // },
+                    // '&:first-of-type > div': {
+                    //   borderTop: 'unset',
+                    // },
+                  }}
+                  content={detailInformation?.faqItems}
+                  accordionHeader={<ZapActionFaqAccordionHeader />}
+                  questionTextTypography="bodyLarge"
+                  answerTextTypography="bodyMedium"
+                  arrowSize={12}
+                />
+              </ZapActionProtocolCard>
+            )}
+            <ZapActionProtocolDisclaimer>
+              <InfoIcon
+                sx={(theme) => ({
+                  color: alpha(theme.palette.text.primary, 0.48),
+                })}
+              />
+              <Typography variant="bodySmall">
+                {market?.attributes.Information}
+              </Typography>
+            </ZapActionProtocolDisclaimer>
+          </ZapProtocolActionInfoBox>
+        </Box>
+        <Box
+          sx={{
+            marginTop: theme.spacing(8),
+            padding: theme.spacing(3, 1),
+            borderRadius: '24px',
+            backgroundColor: theme.palette.surface1.main,
+            boxShadow:
+              theme.palette.mode === 'light'
+                ? '0px 2px 4px rgba(0, 0, 0, 0.08), 0px 8px 16px rgba(0, 0, 0, 0.08)'
+                : '0px 2px 4px rgba(0, 0, 0, 0.08), 0px 8px 16px rgba(0, 0, 0, 0.16)',
+          }}
+        >
+          <Tabs
+            data={tabs}
+            value={tab}
+            ariaLabel="zap-switch-tabs"
+            containerStyles={containerStyles}
+            tabStyles={tabStyles}
+          />
+          {tab === 0 ? (
```

Comment:
A switch would be better, else case is not needed

Reply:
Alright, added a switch statement 
<img src="https://media4.giphy.com/media/KptaYW4AuV2vSkn0Bh/giphy.gif"/>